src/doc/man/generated_txt/cargo-check.txt

```diff
@@ -133,11 +133,12 @@ OPTIONS
            Do not activate the default feature of the selected packages.
 
    Compilation Options
-       --target triple
+       --target triple...
```

Comment:
do we normally do `...` to represent multiple occurrences of a flag?  The first flag I thought of that supports that is `--features` and it doesn't seem to.

Reply:
`--package` and `--example` do that, though it seems inconsistent. I am happy to not add triple dots. At least that can reduce code complexity 😂